sections/06_discussion.md

```diff
@@ -2,57 +2,78 @@
 
 Despite the disparate types of data and scientific goals in the learning tasks
 covered above, several challenges are broadly important for deep learning in the
-biomedical domain.  Here we examine these factors that may impede further
+biomedical domain. Here we examine these factors that may impede further
 progress, ask what steps have already been taken to overcome them, and suggest
 future research directions.
 
-### Evaluation
-
-There are unique challenges to evaluating deep learning predictions in the
-biomedical domain. We focus on TF binding prediction as a representative task to
-illustrate some of these issues. The human genome has 3 billion base pairs, and
-only a small fraction of them are implicated in specific biochemical activities.
-As a result, classification of genomic regions based on their biochemical
-activity results in highly imbalanced classification.  Class imbalance also
-arises in other problems we review, such as virtual screening for drug
-discovery. What are appropriate evaluation metrics that account for the label
-imbalance? The classification labels are formulated based on continuous value
-experimental signals. Practitioners must determine an appropriate procedure for
-formulating binary classification labels based on these signals. In addition,
-the experimental signals are only partially reproducible across experimental
-replicates.  An appropriate upper bound for classification performance must
-account for the experimental reproducibility.
+### General Modeling and Evaluation Considerations
```

Comment:
Fair point, these are all overview-ish topics; on one hand it is kind of artificial to have a title, but I was also trying not to add more sections. Could it work to have the 3 paragraphs (possibly with some additional tweaking) under the top-level Discussion heading?

If we keep it as a separate section, another option is to modify the title to something like "General Optimization and Evaluation Considerations" to try to have it represent a more coherent concept. Alternatively could move that last paragraph out to a subsequent section as it was previously.